packages/@ourworldindata/grapher/src/dataTable/DataTable.tsx

```diff
@@ -714,16 +718,17 @@ export class DataTable extends React.Component<{
     }
 
     @computed get displayDimensions(): DataTableDimension[] {
+        const { entityCount } = this
         // Todo: for sorting etc, use CoreTable?
         return this.columnsWithValues.map((d) => ({
             // A top-level header is only sortable if it has a single nested column, because
             // in that case the nested column is not rendered.
             sortable: d.columns.length === 1,
             columns: d.columns.map((column) => ({
                 ...column,
-                // All columns are sortable for now, but in the future we will have a sparkline that
-                // is not sortable.
-                sortable: true,
+                // All columns with more than one value are sortable for now, but in the future
+                // we will have a sparkline that is not sortable.
```

Comment:
I would actually remove the comment while we're at it; I don't think I know of any future plans to add a sparkline.